python/pylibcugraph/pylibcugraph/graphs.pyx

```diff
@@ -114,6 +117,11 @@ cdef class SGGraph(_GPUGraph):
         match the ordering of the src/dst/edge_id arrays.  Optional (may be
         null).  If provided, edge_id_array must be provided.
     
+    input_array_format: str, optional (default='COO')
+        Input representation used to construct a graph
+            COO: edgelist represented as src_array and dst_array
+            CSR: edgelist represented as offset_array and index_array
```

Comment:
I tend to associate "edgelist" with COO, so maybe that word shouldn't be used for CSR? Either way, this avoids the issue:
```suggestion
            COO: arrays represent src_array and dst_array
            CSR: arrays represent offset_array and index_array
```